homeassistant/components/huawei_lte/switch.py

```diff
@@ -40,30 +41,30 @@ class HuaweiLteBaseSwitch(HuaweiLteBaseEntity, SwitchEntity):
     def _turn(self, state: bool) -> None:
         raise NotImplementedError
 
-    def turn_on(self, **kwargs):
+    def turn_on(self, **kwargs: Any) -> None:
         """Turn switch on."""
         self._turn(state=True)
 
-    def turn_off(self, **kwargs):
+    def turn_off(self, **kwargs: Any) -> None:
```

Comment:
maybe:
```suggestion
    def turn_off(self, **kwargs: Dict[str, Any]) -> None:
```
same for `turn_on`

Reply:
I don't think that's necessary. Keyword arguments always come as str keyed dicts, and although I'm unable to locate a doc reference to back it up, I think `**kwargs: Any` along with some type checker magic is a shorthand notation for that.

Anyway, `turn_on` and `turn_off` are explicitly hinted with `**kwargs: Any' in `ToggleEntity` where they originate from, and like in the `cleanup` case above, we're not affecting in any way what actually gets passed to them as parameters, so copying the superclass hints is the right thing to do.

Reply:
Found the missing doc reference: https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values
> def foo(*args: str, **kwds: int): ...
> In the body of function foo, the type of variable args is deduced as Tuple[str, ...] and the type of variable kwds is Dict[str, int].

Reply:
Thanks for all the explanations and docs reference!